mappings/net/minecraft/block/impl/BlockFire.mapping

```diff
@@ -7,10 +7,12 @@ CLASS none/amp net/minecraft/block/impl/BlockFire
 	FIELD f UP Lnone/asv;
 	METHOD a deserializeState (I)Lnone/asj;
 	METHOD a getDropAmount (Ljava/util/Random;)I
+	METHOD a setBurnLevels (Lnone/akw;II)V
 	METHOD a getCollisionBox (Lnone/asj;Lnone/aiy;Lnone/cn;)Lnone/bcp;
 	METHOD b createStateFactory ()Lnone/ask;
 	METHOD b randomTick (Lnone/aiu;Lnone/cn;Lnone/asj;Ljava/util/Random;)V
 	METHOD c getRenderingState (Lnone/asj;Lnone/aiy;Lnone/cn;)Lnone/asj;
+	METHOD e init ()V
```

Comment:
This doesn't init BlockFire, it inits burn values - initX, where X is the renamed setBurnLevels' latter half, would fit better.